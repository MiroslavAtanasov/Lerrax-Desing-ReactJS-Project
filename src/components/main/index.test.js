import React from 'react'
import renderer from 'react-test-renderer'
import Main from '.'
import TestingEnvironment from '../../test-utils/router'

jest.mock('../mainImg', () => 'MainImg');


describe('Main Component', () => {
    it('should render Main component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    loggedIn: true,
                    id: '123'
                }
            }}>
                <Main />
            </TestingEnvironment>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})