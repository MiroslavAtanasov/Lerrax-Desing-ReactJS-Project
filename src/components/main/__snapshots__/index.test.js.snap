// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Component should render Main component 1`] = `
<div
  className="container"
>
  <MainImg />
  <p
    className="p"
  >
    Popular Across Site
  </p>
  <div />
  <section
    className="section"
  >
    <div
      className="sectionImage"
    >
      <img
        alt="sectionImage2"
        className="image"
        src="03232020-gnp-rugs-694x447_editorial1.jpg"
      />
    </div>
    <div
      className="section2"
    >
      <h1>
        Find Your Perfect Table Size
      </h1>
      <p>
         Before you start shopping, you need to know two things: how to choose the right table size and how to place it in your room. The perfect table can breathe life into a space and unify your furnishings, so itâ€™s important to get it right.
      </p>
      <button
        className="button"
        onClick={[Function]}
      >
        Read More
      </button>
    </div>
  </section>
  <section
    className="section"
  >
    <div
      className="section3"
    >
      <h1>
        Tips on Buying the Best Outdoor Bench
      </h1>
      <p>
         To ensure you score the perfect combination of durability and style, follow our tips on buying outdoor benches. You're sure to find a sturdy all-weather bench that blends seamlessly with your patio's decor.
      </p>
      <button
        className="button"
        onClick={[Function]}
      >
        Read More
      </button>
    </div>
    <div
      className="sectionImage"
    >
      <img
        alt="sectionImage"
        className="image"
        src="03232020-gnp-rugs-694x447_editorial2.jpg"
      />
    </div>
  </section>
</div>
`;
